src/content/zh-cn/fundamentals/performance/index.md

```diff
@@ -0,0 +1,96 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Improving performance starts with minimizing, or at least, optimizing the data that users download. Understanding how a browser renders those resources is a prerequisite for improving code efficiency. After improving it, you need a way to test it. 
+
+{# wf_updated_on: 2017-02-22 #}
+{# wf_published_on: 2017-02-22 #}
+
+# 性能表现 {: .page-title }
+
+提高性能表现的过程要从最小化用户下载数据，或者至少从优化用户下载数据开始。提高代码效率的前提是要理解浏览器是如何渲染这些文件的。最后，你需要一些方法来测试。
```

Comment:
Thanks for reviewing. Should I change here? I think it's not longer than the original text though